docs/conf.py

```diff
@@ -55,15 +55,15 @@
 
 # General information about the project.
 project = 'NI Modular Instruments Python API'
-copyright = '2017-2018, National Instruments'
```

Comment:
We should consider querying and using the current year for the second value.

Reply:
We do. This is a generated file. From the mako template:
```
copyright = '2017-${current_year}, National Instruments'
```